performance-test/scripts/SignUp_Test_Script.jmx

```diff
@@ -0,0 +1,1616 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
```

Comment:
"testdata_reset_password.csv" - not sure what convention is followed here

Reply:
@vishwa-vyom .. I have renamed to testdata_reset_password.txt.